crates/toolchain/transpiler/Cargo.toml

```diff
@@ -15,6 +15,7 @@ eyre.workspace = true
 thiserror.workspace = true
 elf = "0.7.4"
 rrs-lib.workspace = true
+rustc-demangle = "0.1.24"
```

Comment:
if this is only used by the "function-span" feature, can you make it optional and enable it only when "function-span" is enabled? just to prevent dependency issues for a non-core functionality